deploy/kubernetes/csi-attacher.yaml

```diff
@@ -66,7 +66,7 @@ spec:
       serviceAccount: csi-attacher
       containers:
         - name: csi-attacher
-          image: csi_images/csi-attacher:latest
+          image: docker.io/k8scsi/csi-attacher:latest
```

Comment:
Does it have to be k8scsi? It sounds more like k8 scsi :) maybe k8s-csi?

Reply:
this is the repo registered by kubernetes-csi team :)

Reply:
Ok, sounds good

Reply:
The commentary explains the change is being made, but not why; could you add a "why" for other readers, @rootfs ?

Reply:
the image comes from [here](https://github.com/kubernetes-csi/external-attacher/blob/master/deploy/kubernetes/statefulset.yaml#L75)

Reply:
...and that's fine, but there's still no explanation of why the image location changed from csi_images/csi-attacher:latest

Reply:
Right, thats a good point. `csi_images` is not at any public image registry, it is a local image used for development. For testing and production, the images have be hosted in a registry.

Reply:
thanks.